src/main/java/io/vertx/proton/impl/ProtonTransport.java

```diff
@@ -100,18 +100,18 @@ private void handleSocketBuffer(Buffer buff) {
 
       switch (eventType) {
       case CONNECTION_REMOTE_OPEN: {
-        connnection.fireRemoteOpen();
+        connection.fireRemoteOpen();
```

Comment:
Maybe, but if its just a rename theres little to look at, whereas here its mixed with 'actual change' and makes looking at both changes slightly more difficult. Even a separate commit would make life easier :)